besu/src/main/java/org/hyperledger/besu/chainimport/Era1BlockImporter.java

```diff
@@ -119,44 +113,25 @@ public void handleExecutionBlockBody(final Era1ExecutionBlockBody executionBlock
           @Override
           public void handleExecutionBlockReceipts(
               final Era1ExecutionBlockReceipts executionBlockReceipts) {
-            receiptsFutures.add(
-                CompletableFuture.supplyAsync(
-                    () -> {
-                      RLPInput input =
-                          new BytesValueRLPInput(
-                              Bytes.wrap(executionBlockReceipts.receipts()), false);
-                      final List<TransactionReceipt> receiptsForBlock = new ArrayList<>();
-                      input.readList(
-                          (in) ->
-                              receiptsForBlock.add(TransactionReceiptDecoder.readFrom(in, false)));
-                      return receiptsForBlock;
-                    }));
+            // Receipts are generated during the import process. Do nothing
```

Comment:
I am wondering if it would be a faster sync to import here rather than generate on the fly, but that can be a separate PR. Happy to keep this simple for now